### PR TITLE
Remove duplicate testOutputsDirectory definition from integration_test package

### PR DESCRIPTION
**Pull request by jakemac53**
I noticed a getter by this name with the same implementation already exists in flutter_driver, with the same implementation as the initializer for this variable. This deletes the variable and instead exports the getter.

Note that technically this could be breaking, because this version is a mutable variable instead of just a getter. But, I think this was just a mistake.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

File: packages/integration_test/lib/integration_test_driver_extended.dart [hunk 1]
```diff
@@ -14,13 +14,7 @@ import 'package:path/path.dart' as path;
 
 import 'common.dart';
 
-/// Flutter Driver test output directory.
-///
-/// Tests should write any output files to this directory. Defaults to the path
-/// set in the FLUTTER_TEST_OUTPUTS_DIR environment variable, or `build` if
-/// unset.
-String testOutputsDirectory =
-    Platform.environment['FLUTTER_TEST_OUTPUTS_DIR'] ?? 'build';
```
Comment (nate-thegrate) [inline packages/integration_test/lib/integration_test_driver_extended.dart:23]: I noticed that the duplicate code was in 2 places: [`integration_test_driver.dart`](https://github.com/flutter/flutter/blob/27d01bee4558ba71f0407384f5e8e320ee8b7cb0/packages/integration_test/lib/integration_test_driver.dart#L17-L23) and [`integration_test_driver_extended.dart`](https://github.com/flutter/flutter/blob/27d01bee4558ba71f0407384f5e8e320ee8b7cb0/packages/integration_test/lib/integration_test_driver_extended.dart#L17). Maybe it'd be ideal to apply this change in both spots.

**Review by nate-thegrate — APPROVED** [review]
This looks great!

File: packages/integration_test/lib/integration_test_driver_extended.dart (hunk 1 above)
Comment (jakemac53) [reply packages/integration_test/lib/integration_test_driver_extended.dart:23]: Good catch, I can do the same for that one

**Comment by stuartmorgan** [comment]
test-exempt: code refactor with no semantic change

(Modulo the change flagged in the PR description; if we assume that's a mistake then there's no change in intended supported behavior.)

**Pull request merged**